services/resourceGroups/models.go

```diff
@@ -0,0 +1,38 @@
+package resourceGroups
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+var resourceGroupsJSON = `
+{
+  "id": "/subscriptions/cc6b141e-6afc-4786-9bf6-e3b9a5601460/resourceGroups/DefaultResourceGroup-EUS",
+  "name": "DefaultResourceGroup-EUS",
+  "location": "eastus",
+  "properties": {
+    "provisioningState": "Succeeded"
+  }
+}
+`
+
+// ResourceGroupsResponse models the response from the Azure API when creating or updating a resourceGroup
+type ResourceGroupsResponse struct {
+	ID         string `json:"id"`
+	Name       string `json:"name"`
+	Location   string `json:"location"`
+	Properties struct {
+		ProvisioningState string `json:"provisioningState"`
+	} `json:"properties"`
+}
+
+// NewSubscriptionResponse takes a string ID and returns a basic ResourceGroupsResponse object
+func NewSubscriptionResponse(subscriptionID string, resourceGroupName string) ResourceGroupsResponse {
```

Comment:
Did we just copy this and not update the func name?

Reply:
Yup! Will fix. It's a dirty stub but it shouldn't be *this* dirty.